.github/workflows/check.yml

```diff
@@ -55,3 +55,22 @@ jobs:
       with:
         name: disks
         path: disks/
+
+  codestyle:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install packages
+      run: |
+        sudo apt install -y nasm gcc g++ gcc-multilib g++-multilib make binutils grub-pc-bin qemu-system-x86 clang-tidy
```

Comment:
Why do need all those deps ?

Reply:
Not all of them are useful, I was trying to install a missing header and I thought one of these packages would install it.
I will remove the unneeded ones.